test/commons/color/get-background-color.js

```diff
@@ -7,19 +7,59 @@ describe('color.getBackgroundColor', function () {
   var origBodyBg;
   var origHtmlBg;
 
+  /**
+   * Assert that two Colors are equal.
+   * @param {axe.commons.color.Color[]} actual
+   * @param {axe.commons.color.Color[]} expected
+   */
+  function assertColorsEqual(actual, expected) {
```

Comment:
We actually don't need this anymore. In [a recent pr](https://github.com/dequelabs/axe-core/pull/3944#discussion_r1137758374) I updated `axe.commons.color.Color` to be a true class so we could do straight deep equal comparisons between two colors

```js
assert.deepEqual(actual, expected)
```

Reply:
Per offline discussion, added `window.chai.config.truncateThreshold = 0;` to `testutils.js` in concert with this update to ensure assertion failure output actually contains all the properties of the color if an assertion failure occurs